OsmAnd/src/net/osmand/plus/MapMarkersHelper.java

```diff
@@ -132,34 +137,37 @@ public int hashCode() {
 			return result;
 		}
 
+		private static final int[] colors = new int[]{
+				R.color.marker_blue,
+				R.color.marker_green,
+				R.color.marker_orange,
+				R.color.marker_red,
+				R.color.marker_yellow,
+				R.color.marker_teal,
+				R.color.marker_purple
+		};
+
+		public static int[] getColors(Context context) {
```

Comment:
This array must be calculated one time only

Reply:
Fixed